### ISSUE TITLE
[$250] Notifications - Notifications not received for new messages in 1:1 DM

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#47916]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.24.0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Email or phone of affected tester (no customers):** applausetester+omteacher4rp@applause.expensifail.com
**Issue reported by:** Applause - Internal Team

## Action Performed:
Precondition: user should be Sigedn In on main and secondary devices
1. [Account A] Open app
2. [Account A]Navigate to LHN
3. [Account B] Send a new message to account A
4. Open Notification center

## Expected Result:
User should receive a notification for the new message

## Actual Result:
Notifications are not received when the user gets a new message

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [ ] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/user-attachments/assets/ff9bd7df-0566-44aa-bf3c-c31be8b651af

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f80f872056070187</li>
        <li>Upwork Job ID: 1828726126952094219</li>
        <li>Last Price Increase: 2024-08-28</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @OfstadC</details>

**Comment by lanitochka17** [comment]
@miljakljajic  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by miljakljajic** [comment]
Unfortunately I don't think notifications-related bugs can be worked on externally.

**Comment by miljakljajic** [comment]
Waiting to be picked up internally

**Comment by miljakljajic** [comment]
Re-assigning for my parental leave. This is waiting for an internal volunteer.

**Comment by OfstadC** [comment]
Looks like this is still waiting on an Internal volunteer - with low priority.

**Comment by OfstadC** [comment]
I didn't get to this last week - I know we are waiting on an Internal volunteer but I want to retest and make sure it's still occurring - hopefully tomorrow

**Comment by OfstadC** [comment]
Not overdue

**Comment by OfstadC** [comment]
I'm not able to reproduce

**Comment by kavimuru** [comment]
Not repro in ND app and Hybrid app

https://github.com/user-attachments/assets/3c3b951e-2988-4475-b827-e7eeede0f1f2


https://github.com/user-attachments/assets/46956c96-8ff6-4236-8bac-f9d079b2276d

**Comment by OfstadC** [comment]
Thanks @kavimuru - going to close this out since it doesn't seem to be occurring anymore

**Comment by IuliiaHerets** [comment]
QA team faced with this issue in the latest build - 9.0.71-0

https://github.com/user-attachments/assets/2fb5dd8a-55d5-48f3-9974-03d06c927e47

**Comment by OfstadC** [comment]
This looks like an older issue. I'll review early next week

**Comment by OfstadC** [comment]
Not able to reproduce

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by OfstadC** [comment]
Back from the Holidays - this needs to be reproduced still

**Comment by OfstadC** [comment]
https://expensify.slack.com/archives/C9YU7BX5M/p1736792483156099

**Comment by kavimuru** [comment]
Not reproducible anymore.

https://github.com/user-attachments/assets/87b32a67-4a12-46ff-b896-413244301fa6
https://github.com/user-attachments/assets/06dc57db-fb24-47cb-9ee2-cf668836b1b6

**Comment by OfstadC** [comment]
Thanks @kavimuru !

**Comment by OfstadC** [comment]
Will try reproducing again myself this week

**Comment by OfstadC** [comment]
Still not able to reproduce
